lib/cog_cmd/ecs/service/create.rb

```diff
@@ -0,0 +1,84 @@
+require 'cog_cmd/ecs/helpers'
+require 'cog_cmd/ecs/service/helpers'
+require 'securerandom'
+
+module CogCmd::Ecs::Service
+  class Create < Cog::Command
+
+    include CogCmd::Ecs::Helpers
+    include CogCmd::Ecs::Service::Helpers
+
+    attr_reader :service_name, :task_definition, :desired_count
+    attr_reader :cluster_name, :role
+
+    def initialize
+      # args
+      @service_name = request.args[0]
+      @task_definition = request.args[1]
+      @desired_count = request.args[2] || 1
+
+      # options
+      @cluster_name = request.options['cluster']
+      @deployment_config = request.options['deployment-config']
+      @load_balancer = request.options['load-balancer']
+      @role = request.options['role']
+    end
+
+    def run_command
+      raise(Cog::Error, "A service name must be specified.") unless service_name
+      raise(Cog::Error, "A task definition must be specified.") unless task_definition
+
+      response.template = 'service_summary'
+      response.content = create_service
+    end
+
+    private
+
+    def create_service
+      client = Aws::ECS::Client.new()
+
+      params = {
+        service_name: service_name,
+        task_definition: task_definition,
+        desired_count: desired_count,
+        client_token: generate_token
+      }
+      params['cluster'] = cluster_name                       if cluster_name
+      params['deployment_configuration'] = deployment_config if deployment_config
+      params['load_balancers'] = [load_balancer]             if load_balancer
+      params['role'] = role                                  if role
```

Comment:
This is pretty minor, but it might make things slightly more maintainable:

```
optional_params = {
  cluster: cluster_name,
  deployment_configuration: deployment_config,
  load_balancers: load_balancer && [load_balancer],
  role: role
}.reject { |_, v| v.nil? }

params.merge!(optional_params)
```

Also worth noting that it's a little weird to mix symbol and string keys here, but not a huge deal.